src/texture.rs

```diff
@@ -124,11 +124,10 @@ pub fn draw_texture_ex(
         h: texture.height(),
     });
 
-    let (w, h) = params
-        .dest_size
-        .map_or((texture.width(), texture.height()), |dst| {
-            (dst.x(), dst.y())
-        });
+    let (w, h) = match params.dest_size{
```

Comment:
You probably want to run `rustfmt` here

Reply:
Sorry I made the adjustment directly on the github page.